GenderPayGap.WebUI/Views/Scope/FinishOutOfScopeJourney.cshtml

```diff
@@ -48,78 +48,23 @@
-                        </td>
-                    </tr>
-                }
-                <tr class="govuk-table__row">
-                    <th class="govuk-table__header" scope="row">
-                        Employer Name
-                    </th>
-                    <td class="govuk-table__cell">
-                        @Model.Organisation.OrganisationName
-                    </td>
-                </tr>
-                <tr class="govuk-table__row">
-                    <th class="govuk-table__header" scope="row">
-                        Reporting Year
-                    </th>
-                    <td class="govuk-table__cell">
-                        @reportingYearsString
-                    </td>
-                </tr>
-                <tr class="govuk-table__row">
-                    <th class="govuk-table__header" scope="row">
-                        New Scope
-                    </th>
-                    <td class="govuk-table__cell">
-                        OutOfScope
-                    </td>
-                </tr>
-                <tr class="govuk-table__row">
-                    <th class="govuk-table__header" scope="row">
-                        Reason
-                    </th>
-                    <td class="govuk-table__cell">
-                        @reason
-                    </td>
-                </tr>
-            </tbody>
-        </table>
-        
+        <h1 class="heading-large">
+            <span class="heading-secondary">@Model.Organisation.OrganisationName</span>
+        </h1>
         <p class="govuk-body">
-            You have told us that your organisation is not in scope of the gender pay gap reporting regulations for the
-            @reportingYearsString reporting year.
-            We will contact you if we need more information.
+            You have acknowledged that your employer is subject the gender pay
+            gap regulations for the @(reportingYearsString) reporting year.
         </p>
-        
+
         <p class="govuk-body">
-            Return to
-            <a href="@Url.Action("ManageOrganisationGet", "ManageOrganisations", new {encryptedOrganisationId = encryptedOrganisationId})"
-               class="govuk-link">
-                @Model.Organisation.OrganisationName
-            </a>
+            <a class="button" href="@Url.Action("ManageOrganisationGet", "ManageOrganisations", new {encryptedOrganisationId = encryptedOrganisationId})" role="button">Return to @Model.Organisation.OrganisationName</a>
         </p>
-        
         <p class="govuk-body">
-            Return to
-            <a href="@Url.Action("ManageOrganisationsGet", "ManageOrganisations")"
-               class="govuk-link">
-                Your Organisations
-            </a>
+            <a class="button" href="@Url.Action("ManageOrganisationGet", "ManageOrganisations", new {encryptedOrganisationId = ViewContext.RouteData.Values["Id"]})" role="button">Return to Manage Employers</a>

```

Comment:
Should we use `govuk-button` instead? The ticket says "all components up to date as per GDS design system (keep an eye on the ‘details’ component, button styles " (I _think_ that class is up to date?)?

Reply:
I've actually put completely the wrong link here, and I think they should be govuk-link according to the designs in the PDF